README.md

```diff
@@ -6,3 +6,4 @@ This GitHub repository is the source for the pages at [https://docs.microsoft.co
 
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
 For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+No Racial discrimination
```

Comment:
This appears to be covered already by the actual Microsoft Open Source Code of Conduct